src/storage/mutate/AddVerticesProcessor.cpp

```diff
@@ -150,10 +150,6 @@ AddVerticesProcessor::addVertices(PartitionID partId,
         for (auto& index : indexes_) {
             if (tagId == index->get_schema_id().get_tag_id()) {
                 auto indexId = index->get_index_id();
-                if (env_->checkIndexLocked(spaceId_, partId, indexId)) {
-                    LOG(ERROR) << "The part have locked";
-                    return folly::none;
-                }
```

Comment:
I feel this is good, why delete from here?

Reply:
It's seems could avoid check status is not locked (such as building) and later move to locked but the process logic is error. Actually, the index status is one way, just move building to locked. We can check `checkIndexLocked` when `checkRebuilding` failed.

Reply:
> It's seems could avoid check status is not locked (such as building) and later move to locked but the process logic is error. Actually, the index status is one way, just move building to locked. We can check `checkIndexLocked` when `checkRebuilding` failed.

got, A very complicated logic. nice job.